test/fixtures/simple_example/variables.tf

```diff
@@ -16,7 +16,7 @@
 
 variable "parent_id" {
   description = "Id of the resource under which the folder will be placed."
-  default = "65552901371"
+  default     = "212830965325"
```

Comment:
This default should be removed entirely; there is no valid default folder ID for all contributors.

Reply:
I'll remove it

Reply:
Done